bucket/reaper.json

```
@@ -32,7 +32,8 @@
     ],
     "installer": {
         "script": [
-            "Move-Item \"$dir\\`$INSTDIR`$_8_\\*\" -Destination \"$dir\"",
+            "New-Item -Path \"$dir\" -ItemType \"directory\" -Name \"InstallData\"",
+            "Move-Item \"$dir\\`$INSTDIR`$_8_\\*\" -Destination \"$dir\\InstallData\"",
```

I really do not understand why this change is needed, when the application works without this change.

```suggestion
            "Rename-Item \"$dir\\`$INSTDIR`$_8_\" 'InstallData'",
```

---

It does if you're upgrading, since necessary content (themes, effects, etc) is already in %APPDATA%/REAPER. With clean install I get the result shown on the first screenshot.

Steps to reproduce:
1. Uninstall reaper
2. Backup and remove %APPDATA%/REAPER directory
3. scoop install reaper
4. Run program

Why?
Reaper setup program has two modes, "ordinary" install and portable.
1. In portable mode, executable and program data is unpacked to a single directory, on the first run reaper.exe checks for reaper.ini (which is created by setup program with "wnd_state=0" content inside), if it's not present, it behaves like an ordinary install - that's the case with reaper.json in master branch.
My first idea was to make configs persistent, but there is much more data which should persist and maintaining it would be hard and risky (imagine loosing hours of work put in presets being lost with a reaper update).
2. In ordinary mode, on the first run, reaper.exe copies content of InstallData subfolder (created by setup program) to %APPDATA%/REAPER. Without InstallData folder (current reaper.json in master) there is no data to copy.

---

Now I think i misundersrtood the question.
"Rename-Item \"$dir\\`$INSTDIR`$_8_\" 'InstallData'"
is fine, but I didn't know this PowerShell command.